Auctionator.lua

```diff
@@ -3287,12 +3287,9 @@ function Atr_HEntryOnClick(self)
     gHentryTryAgain = nil;
   end
 
-  local _, itemLink;
-  local IDstring;
   local entryIndex = line:GetID();
-
-  itemName = gHistoryItemList[entryIndex].name;
-  itemLink = gHistoryItemList[entryIndex].link;
+  local itemName = gHistoryItemList[entryIndex].name;
+  local itemLink = gHistoryItemList[entryIndex].link;
```

Comment:
_ and IDstring are unused, can move the local to itemName and itemLink which is fine